uli-website/src/pages/about.mdx

```diff
@@ -5,45 +5,37 @@ import MultiLingualAbout from "../components/atoms/MultiLingualAbout";
 Uli was born from the collective labour of journalists, activists, community influencers, and writers engaged in the struggle against the interwoven caste, religion, gender and sexuality-based violence both online and offline. They contributed towards the development of the Uli tool that allows us to filter out offensive words/phrases, archive problematic content and call out for support. Uli is not imagined as the final product, but rather a simple tool, a chisel, that allows one to make a room of one’s own or courtyards where people can come together, share stories, and reflect on the future we all want to see.
-In its second year, Uli is evolving as a user-facing tool; as well as a set of resources that researchers, Trust and Safety teams and professionals, and journalists can use in their workflows. 
+In its second year, Uli is evolving as a user-facing tool; as well as a set of resources that researchers, Trust and Safety teams, professionals, and journalists can use in their workflows.
 
-### (Uli in Media)[/uli_media]
+## (Uli in Media)[/uli_media]
 
-
-## The Team 
+## The Team
 
 Uli was launched in 2021 as a joint project between The [Centre for Internet and Society (CIS)](https://cis-india.org/) and [Tattle Civic Tech](https://tattle.co.in/). It is currently hosted at Tattle and is being developed in close partnership with feminist organizations in India. You can read more about the people involved in the pilot here: [Uli Pilot Team](https://uli.tattle.co.in/team-2021/)
 
-
 The current project team is comprised of:
 
-  * **Denny George** : Engineering lead, [denntenna](https://denntenna.github.io/),
-  * **Siddharth Manohar**: Data Governance Researcher,
-  * **Sudeep Duggal** : Fullstack engineer,
-  * **Tarunima Prabhakar** : Project Manager, [website](https://tarunima.com),
-  * **Yash Budhwar** : Operations and contracts,
-  * **Yohan Mathew** : ML DevOps  
-
-  Advisors/ Volunteers:
-  * **Ambika Tandon** : Overall Project,
-  * **Arnav Arora** : Data Science, [@rnav_arora](https://twitter.com/rnav_arora),
-  * **Cheshta Arora** : Researcher,
-  * **Kaustubha Kalidindi** : Community Building,
- 
- For our work in 2023 we have partnered with four stellar organizations:
- 1. Chambal Media Private Limited [(Khabar Lahariya)](https://khabarlahariya.org/about-us/)
- 2. [Point of View](https://pointofview.org/)
- 3. [Nazariya: Queer Feminist Resource Group](https://thenazariyafoundation.org/)
- 4. [Feminism in India](https://feminisminindia.com/) (FII)
+  * **Denny George** : Engineering lead, [denntenna](https://denntenna.github.io/)
+  * **Aatman Vaidya** : Software Developer
+  * **Maanas B** : Software Developer
+  * **Tarunima Prabhakar** : Project Manager, [website](https://tarunima.com)
+  * **Kaustubha K** : Counsel, Program Manager - Uli
```

Comment:
@Mahimahto you can add your name here too. You can put the designation as Software Developer